bucket/np21w.json

```diff
@@ -36,14 +32,17 @@
     "bin\\np21w.ini",
     "bin\\np21x64w.ini"
   ],
+  "bin": [
+    "bin\\np21w.exe"
+  ],
   "shortcuts": [
     [
-      "bin\\np21x64w.exe",
-      "Neko Project 21／W"
```

Comment:
`／` is hard to type on non-JIS environment and `/` does not work, so I removed it.